lib/nerves_motd.ex

```diff
@@ -31,70 +33,113 @@ defmodule NervesMOTD do
   @spec print([option()]) :: :ok
   def print(opts \\ []) do
     {:ok, _} = Application.ensure_all_started(:nerves_runtime)
-    IO.puts(generate(opts))
+
+    IO.puts(logo(opts))
+
+    IO.puts(uname())
+
+    Enum.each(rows(), &IO.puts(["  ", format_row(&1)]))
+
+    IO.puts("""
+
+    Nerves CLI help: https://hexdocs.pm/nerves/using-the-cli.html
+    """)
```

Comment:
```suggestion
   [
     logo(opts),
     uname(),
     "\n\n",
     Enum.map(rows(), &format_row/1),
     """
     Nerves CLI help: https://hexdocs.pm/nerves/using-the-cli.html
     """
   ]
   |> IO.puts()
```

This requires adding a newline to the end of the log and new lines in `format_row/1` as well since `IO.puts/1` was adding them. 

I kind of like how this looks even though normally I shy away from 1 item pipelines. It could be that we add a second function that returns the MOTD as iodata and then call it from `print/1`. I'm also ok with deferring to a separate PR, but figured that I'd mention.

Reply:
Sounds good. Let's do it in this PR.